README.md

```diff
@@ -157,5 +157,5 @@ instructions there.
 
     # This will expose UAA for use; adjust the contents if you're not deploying
     # to vagrant
-    kubectl create -n uaa -f kube-test/exposed-ports.yml
+    kubectl create -n uaa
```

Comment:
Actually this should go away, since you're now creating nothing (and `kubectl` will either get confused or ask for stuff on stdin, not sure which).